activerecord/lib/rails/generators/active_record/model/templates/model.rb

```diff
@@ -1,7 +1,7 @@
 <% module_namespacing do -%>
 class <%= class_name %> < <%= parent_class_name.classify %>
 <% attributes.select(&:reference?).each do |attribute| -%>
-  belongs_to :<%= attribute.name %><%= ', polymorphic: true' if attribute.polymorphic? %><%= ', required: true' if attribute.required? %>
```

Comment:
Since you can opt out of this behaviour with `config.active_record.belongs_to_required_by_default` we should probably keep this in, It also looks like you can do either `required: false` or `optional: true` so nothing seems fully deprecated yet.